client/blocks/inline-help/style.scss

```diff
@@ -249,6 +249,15 @@
 	}
 }
 
+//...but not on the Preview page.
+.is-secondary-view-active .inline-help__checklist + .checklist-navigation {
+	display: block;
```

Comment:
I think it would be better to control visibility by add/removing the components using React rather than having display logic in the CSS.

Reply:
Agreed! This has already been handled by #29418, I just need to rebase this PR with master.